### PR TITLE
Add a contact page

### PR DESCRIPTION
**Pull request by brianwarner**
Signed-off-by: Brian Warner <brian@bdwarner.com>

### CODE REVIEW

File: content/contact.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+To report a technical issue with GraphQL, please file an issue on [GitHub](https://github.com/graphql).
+
+## General
+
+For all other inquiries, please email [info@graphql.org](mailto:info@graphql.org).
+
+Our mailing address is:
+
+**GraphQL Foundation**
+1 Letterman Drive, Building D, Suite D4700
```
Comment (lucperkins) [inline content/contact.md]: Add a blank line here, or else these two lines are squashed into one 😄

Comment (brianwarner) [reply content/contact.md]: Thanks @lucperkins!

**Comment by brianwarner** [comment]
LGTM, that's as many tweaks as I'm going to make for now.  @caniszczyk and @lucperkins, this is ready for final review for merging.

**Comment by caniszczyk** [comment]
LGTM you may want to embed a calendar for the GraphQL WG meetings

**Pull request merged**